Yoast/wordpress-seo
# Remove first usable image fallback from opengraph

## Summary

This PR can be summarized in the following changelog entry:

* _N/A_ Remove opengraph fallback to images from the content.

## Relevant technical choices:

* We might want to remove `WPSEO_Content_Images`.
* We might want to provide a code snippet for users on our knowledge base that still want this functionality.

## Test instructions

This PR can be tested by following these steps:

* To reproduce the old situation create a new post, add an image to the post. 
* Make sure you have not set a:
-Featured image for the post
-OG image (under Social tab -> Facebook image)
-Default og image (under SEO -> Social -> Facebook)
All of these should be empty. 
* Publish the post, view it and check source. You will find the image you put in the post there in the `og:image` tag. It looks like this: 
```
<meta property="og:image" content="http://local.wordpress.test/wpcontent/uploads/2018/02/20141213133819-3-1024x683.jpg" />
<meta property="og:image:width" content="1024" />
<meta property="og:image:height" content="683" />
```
There should also be a twitter image: 
```
<meta name="twitter:image" content="http://local.wordpress.test/wp-content/uploads/2018/05/images-1.jpeg" />
```

* Checkout this branch. Reload the page, the og:image tags should be gone. Also, the twitter:image should be gone. 
* Set a og:image through one of the other methods (for example featured image) to make sure this functionality still works. 

## Quality assurance


Fixes #11262
